l2geth/rollup/sync_service.go

```diff
@@ -75,6 +76,8 @@ type SyncService struct {
 	feeThresholdDown               *big.Float
 	cfg                            Config
 	applyLock                      sync.Mutex
+	updateGasPriceTxPool           chan *types.Transaction
+	verifiedMap                    map[common.Hash]*big.Int
```

Comment:
Use `sync.Map`, concurrent issue must be taken into consideration.

Reply:
you are right